### DISCUSSION TITLE
Yaml Validation using schema

### DISCUSSION THREAD

**Discussion by ChetanKB** [redhat-developer/vscode-yaml#792] [General]
Hi all,

I have a YAML file and JSON schema file, I need to validate the YAML file against that JSON schema, I also want to compare the YAML file and check the YAML Validation. How can I use the yaml-language-server in the command-line interface?
** If this is not possible please could you suggest any other tool with that I can do the YAML validation and comparison with JSON schema.**

**Comment by gorkem** [comment]
We do not provide a CLI for `yaml-language-server` but I have seen GitHub action implementations by various people that wrap the language server. For instance, [this one](https://github.com/nwisbeta/validate-yaml-schema) may give hints on how to achieve command-line usage.
